src/routes/dbInit/__mocks__/mockAllDocs.ts

```diff
@@ -0,0 +1,126 @@
+// tslint:disable
```

Comment:
I find this a little bit concerning. What issues are caused if you remove this line? I think there is probably a way to refactor to remove this line and get it to lint.